src/main/webapp/college-info.js

```diff
@@ -430,7 +430,9 @@ function loadLogin() {
         } else {
           // Sets login link.
           logLink.setAttribute('href', loginObj.loginURL);
-          logLink.innerText = 'Sign in here to leave a comment.';
+          const logIMG = document.createElement('IMG');
+          logIMG.setAttribute('src', 'https://developers.google.com/identity/images/btn_google_signin_light_normal_web.png');
```

Comment:
Nit: usually it is better to store the static image in our site/local, because Google might change it without notifying us :). It is fine to have the url for now.